SAR_Training/English/Master/ISCE3_Sentinel1_RTC.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    if scene[:3] == "S1A":
</pre><p>this is kind of a stylistic thing, but this could be refactored to <code>platform = scene[:3] if any([n == scene[:3] for n in ["S1A", "S1B"]) else None</code>, then the <code>else</code> becomes <code>if not platform:</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFOpenSARlab/opensarlab-notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>